src/components/ForecastDetails.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/ForecastDetails.css";
+
+function ForecastDetails({ forecast }) {
+  const { date, humidity, temperature, wind } = forecast;
+  const options = { weekday: "long", month: "long", day: "numeric" };
+  const formattedDate = new Date(date).toLocaleDateString("en-GB", options);
+
+  return (
+    <div className="forecast-details" data-testid="forecast-details">
+      <div className="forecast-details__date">
+        <h2 className="forecast-details__date">{formattedDate}</h2>
+      </div>
+      <div
+        className="forecast-details__temperature"
+        data-testid="temperature-div"
+      >
+        <div
+          className="forecast-details__max-temperature"
+          data-testid="temperature-div__max"
+        >
+          High: {temperature.max}
+          &deg;C
+        </div>
+        <div
+          className="forecast-details__min-temperature"
+          data-testid="temperature-div__min"
+        >
+          Low: {temperature.min}
+          &deg;C
+        </div>
+      </div>
+      <div className="forecast-details__humidity">Humidity: {humidity}%</div>
+      <div className="forecast-details__wind">
+        Wind: {wind.speed}mph {wind.direction}
```

Comment:
In your app, the wind direction comes up as an abbreviation like 'wsw'. This isn't very readable for the user. Is there any way we can display something else for the user? How can we adjust our code to do this? Message me on slack if you're not sure and I can talk it through with you